orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.groovy

```diff
@@ -32,6 +32,8 @@ interface ExecutionRepository {
   void cancel(String id)
   void cancel(String id, String user)
 
+  void forceCancel(String id, String user)
```

Comment:
The original `cancel` method allows Spring Batch to cleanly finish the job — i.e. it flags it for cancellation. The `forceCancel` method kills one that's no longer running.